Tests/Utils/XCTests.swift

```diff
@@ -98,3 +98,10 @@ extension SequenceType {
         return dict
     }
 }
+
+extension XCTest {
+    var is32Bit: Bool {
+        let is32bit = sizeof(Int) == sizeof(Int32)
+        return is32bit
```

Comment:
This would be more concise/less repetitive if it was just returning the expression